imap_processing/codice/constants.py

```diff
@@ -257,9 +282,10 @@
     },
     CODICEAPID.COD_LO_NSW_PRIORITY_COUNTS: {
         "dataset_name": "imap_codice_l1a_lo-nsw-priority",
-        "dims": {"spin_sector": 12, "esa_step": 128},
+        "input_dims": {"esa_step": 128, "spin_sector": 12},
         "instrument": "lo",
         "num_counters": 2,
+        "output_dims": {"spin_sector": 12, "esa_step": 128},
```

Comment:
In the meeting we discussed that the "esa_step" should be the first dimension here I thought, so similar to what the input_dims is for this case.

I'm wondering if most of these output_dims are only needed for the test comparisons with the other CDFs and not necessarily for the actual production products?

Reply:
Yes, exactly. These will eventually change to match the conventions we settled on during the STM. Joey is working on getting me updated validation data to reflect this, but for now I will keep it like this to match the current validation data.